dc-cudami-server/dc-cudami-server-business/src/main/java/de/digitalcollections/cudami/server/business/impl/service/security/UserServiceImpl.java

```diff
@@ -109,4 +113,13 @@ public User update(User user, Errors results) {
   public List<User> findActiveAdminUsers() {
     return userRepository.findActiveAdminUsers();
   }
+
+  private void setDefaultSorting(PageRequest pageRequest) {
+    if (pageRequest.getSorting() == null
```

Comment:
vielleicht können wir das ja in den pagerequest als methode "hasSorting" auslagern?

Reply:
die drei ifs....

Reply:
Ist eingebaut :+1: